appgate/__main__.py

```diff
@@ -291,7 +291,10 @@ def main_api_info(spec_directory: Optional[str] = None) -> None:
 
 
 def main_dump_crd(
-    stdout: bool, output_file: Optional[str], spec_directory: Optional[str] = None
+    stdout: bool,
```

Comment:
For the future, it is appreciated if you do actual changes in a commit and run black in a separate later commit.